### PR TITLE
Add RabbitMQ automated support.

### PR DESCRIPTION
**Pull request by Crell**
Resolves #27 

Rewritten based on the prior PR.

### CODE REVIEW

File: tests/FlexBridgeRabbitMqTest.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    'query'    => [],
+                ]
+            ]
+        ];
+
+        $this->defaultValues = [];
+
+    }
+
+    public function testNotOnPlatformshDoesNotSetDatabase(): void
```
Comment (botris) [inline tests/FlexBridgeRabbitMqTest.php]: typo, database should probably be rabbit.

File: tests/FlexBridgeRabbitMqTest.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        unset($rels['rabbitmqqueue']);
+
+        putenv(sprintf('PLATFORM_RELATIONSHIPS=%s', base64_encode(json_encode($rels))));
+
+        mapPlatformShEnvironment();
+
+        $this->assertArrayNotHasKey('MESSENGER_TRANSPORT_DSN', $_SERVER);
+    }
+
+    public function testElasticsearchRelationshipSet(): void
```
Comment (botris) [inline tests/FlexBridgeRabbitMqTest.php]: typo, elastic should be rabbit

**Review by botris — COMMENTED** [review]
some typo's

**Review by botris — COMMENTED** [review]
not working for me:
Could not connect to the AMQP server. Please verify the provided DSN.

File: tests/FlexBridgeRabbitMqTest.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        $this->relationships = [
+            'rabbitmqqueue' => [
+                [
+                    'scheme'   => 'amqp',
+                    'username' => 'guest',
+                    'password' => 'guest',
+                    'host'     => 'rabbitmq.internal',
+                    'port'     => 5672,
+                    'path'     => null,
+                    'query'    => [],
```
Comment (botris) [inline tests/FlexBridgeRabbitMqTest.php:28]: vhost is missing

Comment (Crell) [reply tests/FlexBridgeRabbitMqTest.php:28]: It's only included on Dedicated environments, actually.  Hence the default of `%2f` hard coded into the formatter.

**Review by botris — APPROVED** [review]
tested on grid, works as expected

File: platformsh-flex-env.php [hunk 4]
```diff
@@ -199,6 +220,23 @@ function mapPlatformShDatabase(string $relationshipName, Config $config) : void
     setEnvVar('DATABASE_URL', $config->formattedCredentials($relationshipName, 'doctrine'));
 }
 
+/**
+ * Maps the specified relationship to the DATABASE_URL environment variable, if available.
```
Comment (chadwcarlson) [inline platformsh-flex-env.php]:
```suggestion
 * Maps the specified relationship to the MESSENGER_TRANSPORT_DSN environment variable, if available.
```

**Pull request merged**